notebooks/features/cognitive_services/CognitiveServices - Overview.ipynb

```diff
@@ -139,31 +174,32 @@
     "from synapse.ml.cognitive import *\n",
     "\n",
     "# A general Cognitive Services key for Text Analytics, Computer Vision and Form Recognizer (or use separate keys that belong to each service)\n",
-    "service_key = find_secret(\"cognitive-api-key\")\n",
+    "service_key = find_secret(\"cognitive-api-key\") # Replace it with your cognitive service key, check prerequisites for more details\n",
     "service_loc = \"eastus\"\n",
     "\n",
     "# A Bing Search v7 subscription key\n",
-    "bing_search_key = find_secret(\"bing-search-key\")\n",
+    "bing_search_key = find_secret(\"bing-search-key\") # Replace it with your cognitive service key, check prerequisites for more details\n",
     "\n",
-    "# An Anomaly Dectector subscription key\n",
-    "anomaly_key = find_secret(\"anomaly-api-key\")\n",
+    "# An Anomaly Detector subscription key\n",
+    "anomaly_key = find_secret(\"anomaly-api-key\") # Replace it with your cognitive service key, check prerequisites for more details\n",
     "anomaly_loc = \"westus2\"\n",
     "\n",
     "# A Translator subscription key\n",
-    "translator_key = find_secret(\"translator-key\")\n",
+    "translator_key = find_secret(\"translator-key\") # Replace it with your cognitive service key, check prerequisites for more details\n",
     "translator_loc = \"eastus\"\n",
     "\n",
     "# An Azure search key\n",
-    "search_key = find_secret(\"azure-search-key\")"
+    "search_key = find_secret(\"azure-search-key\") # Replace it with your cognitive service key, check prerequisites for more details"
```

Comment:
Maybe make this warning something clearer like 

# Replace the call to find_secret with your key as a python string. e.g. key="27snaiw..."